Tests/LeafKitTests/TagTests.swift

```diff
@@ -175,7 +175,11 @@ class TagTests: XCTestCase {
         The date is 2020-11-09T14:30:00
         """
 
-        let now = 1604932200 - Calendar.current.timeZone.secondsFromGMT()
+        let formatter = DateFormatter()
```

Comment:
Not really necessary but I'd cache the formatter

Reply:
For the purposes of a single use by each of four tests? Hardly worth it.